index.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>YouTube Clone</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css">
+    <script src="https://kit.fontawesome.com/d7c2fd4a1e.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="./css/style.css">
+</head>
+
+<body>
+    <header>
+        <nav id="grid-container-header">
+            <div class="logo-container">
+                <span class="black"><i class="fa fa-bars" aria-hidden="true"></i></span>
+                <i class="fa fa-youtube-play fa-lg" aria-hidden="true"><span class="black">YouTube Clone</span></i>
+            </div>
+            <div class="search-container">
+                <input type="text" name="Search Bar" id="text-field" placeholder="Search">
+                <button><i class="fa fa-search" aria-hidden="true"></i></button>
+            </div>
+            <div class="icon-container">
+                <div class="right">
+                    <i class="fab fa-adn size"></i>
+                </div>
+                <div class="right">
+                    <i class="fas fa-bell size"></i>
+                </div>
+                <div class="right">
+                    <i class="fas fa-th size"></i>
+                </div>
+                <div class="right">
+                    <i class="fas fa-video size"></i>
+                </div>
+            </div>
+        </nav>
+    </header>
+    <main id="grid-container-main">
+        <div id="section-container">
+            <section id="section-container-main">
+                <div>
+                    <iframe src="https://www.youtube.com/embed/WNOBl1mtsMY"
+                        allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+                        allowfullscreen></iframe>
+                </div>
+                <h1>Cute and Funny Dog Compilation</h1>
+                <div id="grid-container-video-main-info">
+                    <p>1,340,783 views • 18 Dec 2018</p>
+                    <div id="grid-container-video-main-info-extra">
+                        <i class="fa fa-thumbs-up" aria-hidden="true"> 100K</i>
+                        <i class="fa fa-thumbs-down" aria-hidden="true"> 110</i>
+                        <i class="fa fa-share"> Share</i>
+                    </div>
+                </div>
+            </section>
+            <hr>
+            <section id="grid-container-video-additional-info">
+                <div id="video-owner">
+                    <i class="fab fa-adn"></i>
+                </div>
+                <div id="video-information">
+                    <h2>Animal World</h2>
+                    <h3>100K Subscribers</h3>
```

Comment:
- [ ] The layout here could be closer to the original. Try to make the font-sizes and colors more similar to the original

| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/90419557-212f9b00-e08d-11ea-945d-b360fdcc221e.png) |![image](https://user-images.githubusercontent.com/61479648/90419589-2d1b5d00-e08d-11ea-95e7-41697775bf1e.png)  |